.github/workflows/update-version.yml

```
@@ -0,0 +1,88 @@
+name: Update WhatsApp Version
+
+on:
+  schedule:
+    # Run on the 1st of every month at 02:00 UTC
+    - cron: '0 2 1 * *'
```

```suggestion
    - cron: '0 2 1 * *'
```

I believe that this should be more frequent. At least once a week or once every 2-3 weeks at most.

This is so every NPM release can have both ver and proto updates.